File: learning/cs50/pset1/mario.c
#include <stdio.h>
#include <cs50.h>

int main (void)
{

    //Get Height from the user
    int n;
    do
    {
        n = get_int("Size: ");
    }
    while(n < 1 || n > 8);

    // Printing squares;
    for (int i = 0; i < n; i++)
    {
        // Blanck :
        for (int j = n - 1; j > i; j--)
        {
            printf(" ");
        }

        // hashes :
        for (int j = 0; j <= i ; j++)
        {
            printf("#");
        }
    printf("\n");
    }
}

File: learning/cs50/pset6/mario/less/mario.py
# Get Height from the user

while True:
    n = int(input("Height: "))
    if (n > 0 and n <=8):
        break

# Printing image

for i in range(n):
    print((n - 1 - i) * " ", end = "")
    print((i+1) * "#")